gcx/styles/src/partials/pixel.hbs

```diff
@@ -0,0 +1,3 @@
+    {{#with site.keys.trackingPixel}}
```

Comment:
```suggestion
    {{#with site.keys.tracking_pixel}}
```

In all of the associated PRs you are using the key `tracking_pixel`, not `trackingPixel`.

Reply:
Thanks for digging into the details! You've actually encountered one of the intricacies of Antora.

From the Antora [docs](https://docs.antora.org/antora/latest/extend/configure-extension/#configuration-key-transformation):

> In YAML, key names use the snake_case naming convention. In JavaScript, property names use the camelCase naming convention. To help bridge the naming convention mismatch between YAML and JavaScript, Antora automatically transforms snake_case key names in the playbook file into camelCase properties on the configuration object.

Perhaps this is just an opinionated stance that the Antora maintainers have taken on naming conventions, because it's my understanding that camelCase is now a common convention in YAML (perhaps it wasn't when the Antora project started). In fact, I initially had defined the key name as `trackingPixel` in the playbook when I was testing this change for the first time. However, Antora wouldn't honor the conditional until I changed the key name to `tracking_pixel`.

In any case (pun intended), Antora's Default UI setup uses the camelCase convention in its handlebars templates, and it uses snake_case for its default playbook keys. So for the time being I think we probably need to follow that convention, whether or not we fully understand why 😕